conans/test/utils/conan_v2_tests.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from conans.client.tools import environment_append
+from conans.test.utils.tools import TestClient
+
+from conans.util.conan_v2_mode import CONAN_V2_MODE_ENVVAR
+
+
+class ConanV2ModeTestCase(unittest.TestCase):
+
+    @staticmethod
+    def get_client(*args, **kwargs):
+        # TODO: Initialize with the default behavior for Conan v2
```

Comment:
I don't see why this is necessary, nothing should be special about the client instance, a totally normal, one, only difference should be CONAN_V2_MODE_ENVVAR

Reply:
Not for these tests, but in the v2 context, this client should enable revisions by default, a different package_id_mode,...

Reply:
I see. Just remember that these tests are now run both with and without revisions, so if the tests enable revisions here by default, they can be ran twice.